Sources/AWSSDKSwiftCore/Encoder/XMLDecoder.swift

```diff
@@ -18,7 +18,7 @@ import struct Foundation.URL
 import class  Foundation.DateFormatter
 
 /// The wrapper class for decoding Codable classes from XMLNodes
-class XMLDecoder {
+public class XMLDecoder {
```

Comment:
we could move this to a separate `AWSXML` target and don't export this as a `library` product. this way we don't expose this to end users and we would keep our api smaller (and less breaking).